Lib/test/test_ast.py

```diff
@@ -331,7 +331,7 @@ def test_base_classes(self):
 
     def test_field_attr_existence(self):
         for name, item in ast.__dict__.items():
-            if isinstance(item, type) and name != 'AST' and name[0].isupper():
+            if isinstance(item, type) and name != 'AST' and "ast" in item.__module__ and name[0].isupper():
```

Comment:
This guard is already too big, I suggest we factor into a function that describes what we are actually checking for?

Reply:
I dont think I can describe what it does but I'll refactor it into a guard function and use it.

Reply:
Something like  `is_valid_test_class` is enough.